### PR TITLE
Moving model to top level

### PR DESCRIPTION
**Pull request by gvegayon**
- Merges `./README.md` with `model/README.md`.
- Moves all the content under `model` to `.`.
- Adjust the paths for GHA to ensure they run under the new project structure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,63 @@
-# Multisignal Renewal Project
+# PyRenew: A Package for Bayesian Renewal Modeling with JAX and Numpyro.
```
Comment (damonbayer) [inline README.md:1]:
```suggestion
# PyRenew: A Package for Bayesian Renewal Modeling with JAX and NumPyro.
```

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -59,7 +59,6 @@ repos:
             "--baseline",
             ".secrets.baseline",
             "--exclude-files",
-            "model/docs/*_cache",
```
Comment (damonbayer) [inline .pre-commit-config.yaml:62]: What does this change do?

**Review by damonbayer — CHANGES_REQUESTED** [review]
@gvegayon Can you also update the installation instructions in `getting_started`?

File: .pre-commit-config.yaml (hunk 2 above)
Comment (gvegayon) [reply .pre-commit-config.yaml:62]: That was to exclude cache data generated by quarto. It was relevant when we had the tutorials under model/docs.

**Comment by gvegayon** [comment]
> @gvegayon Can you also update the installation instructions in `getting_started`?

Good catch! Just did. Should be good now.

**Comment by gvegayon** [comment]
Codecov is not uploading, so I'm guessing it's an issue on their side. All other test are passing.

**Comment by damonbayer** [comment]
@gvegayon Can you remove the `pipeline` github actions?

**Pull request merged**